src/types/icon.ts

```diff
@@ -0,0 +1 @@
+export type Icon = 'xmark' | 'shopping-bag' | 'check-square';
```

Comment:
If you find yourself doing something like this ask yourself "would every person using this library have to do this?" If the answer is yes then the library probably exports the type you're after